src/main/java/me/zeroeightsix/kami/module/modules/chat/ChatFilter.kt

```diff
@@ -60,14 +60,14 @@ class ChatFilter : Module() {
         return Pattern.compile(filter, Pattern.CASE_INSENSITIVE).matcher(message).find()
     }
 
-    public override fun onEnable() {
+    override fun onEnable() {
         val bufferedReader: BufferedReader
         try {
             sendChatMessage("$chatName Trying to find '&7chat_filter.txt&f'")
             bufferedReader = BufferedReader(InputStreamReader(FileInputStream("chat_filter.txt"), "UTF-8"))
-            var line: String
+            var line = bufferedReader.readLine()
```

Comment:
will this work? this will only read the first line?

Reply:
yes it reads the first line and if it is null (ex. empty file), then it will skip the while loop. 
Just like spammer [](https://github.com/kami-blue/client/blob/d830832c11063c9d4f818fd0e1535a273f1e245f/src/main/java/me/zeroeightsix/kami/module/modules/chat/Spammer.kt#L39-L44)